.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
What this .idea folder is? Is it the folder used by the IDE you're developing this? If this is the case, can you please add the .idea folder under the main .gitignore file and remove the folder from the repo? This comment is for every entry we see below that is related to anything inside .idea

Reply:
yes it is added by the IDE